script.js

```diff
@@ -0,0 +1,68 @@
+const johnTeamScores = [89, 20, 103];
+const mikeTeamScores = [116, 94, 123];
+const johnsAvrg = calcAverage(johnTeamScores);
+const mikesAvrg = calcAverage(mikeTeamScores);
+const sameAvrg = ' have the same score';
+
+const higherAverage = {
+    winner: 'John and Mike',
+    highestAvrg: '\'s team has the highest average score of ',
+    score: '',
+
+    setWinner() {
+
+        if (johnsAvrg !== mikesAvrg && johnsAvrg > mikesAvrg) {
+            this.winner = 'John';
+            this.score = johnsAvrg;
+        }
+        else if (johnsAvrg !== mikesAvrg) {
+            this.winner = 'Mike';
+            this.score = mikesAvrg;
+        }
+        else {
+            this.highestAvrg = sameAvrg;
+        }
+    }
+};
+
+higherAverage.setWinner();
+
+function calcAverage(scores) {
```

Comment:
Hello can we add some documentation for the code [here](https://devhints.io/jsdoc) and [here](https://jsdoc.app/index.html) are some examples